docs/modules/ROOT/pages/how-tos/migrate-to-lokistack.adoc

```diff
@@ -0,0 +1,96 @@
+= Migrate to Lokistack
```

Comment:
For the whole PR:
```suggestion
= Migrate to LokiStack
```

Reply:
I don't know... that looks wrong, even if RedHat uses this naming. Also I think we called it Lokistack everywhere else.

Reply:
Looking through https://github.com/grafana/loki the most common case is LokiStack, but I think I've seen all of:
```
LokiStack
lokiStack
Lokistack
lokistack
Locki Stack
```

nothing that needs to be changed...